torch/fx/proxy.py

```diff
@@ -3,12 +3,57 @@
 import inspect
 import operator
 
-from .graph import magic_methods, reflectable_magic_methods
-from typing import TYPE_CHECKING, Tuple, Dict, Optional, Iterable, NoReturn, Any
-from .node import Target, Node
-
-if TYPE_CHECKING:
-    from .symbolic_trace import DelegateBase
+from .graph import magic_methods, reflectable_magic_methods, Graph
+from typing import Tuple, Dict, Optional, Iterable, NoReturn, Any, Union, Callable
+from .node import Target, Node, Argument, base_types
+
+class TracerBase:
```

Comment:
Kinda weird this is in `proxy.py`. Can it be in its own file?

Reply:
I tried but it's cyclic: TracerBase depends on Proxy and Proxy depends on TracerBase.